admin/ci-tools/cleanup-vagrant-boxes

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+"""
+A tool for use by the CI system to clean up old versions of Vagrant boxes that
+get downloaded as part of CI work.  This helps keep disks of slaves from
+filling up with useless old boxes and eventually failing when there are too
+many.
+
+The newest version of the box is kept.
+
+Usage::
+
+  cleanup-vagrant-boxes <name of box>
+"""
+
+from __future__ import unicode_literals, print_function
+
+from distutils.version import LooseVersion
+from subprocess import check_output
+
+
+class Box(object):
```

Comment:
:question: Docstring? (I don't think it really needs one, but AIUI we do insist on one for all classes & methods).